CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2201,6 +2209,15 @@ protected override void OnMouseUp(MouseButtonEventArgs e)
         {
             OnMouseButton(e);
 
+            if (e.ChangedButton == MouseButton.Left && e.LeftButton == MouseButtonState.Released)
+            {
+                //Release the mouse capture that we grabbed on mouse down.
+                //We won't get here if e.g. the right mouse button is pressed and released
+                //while the left is still held, but in that case the left mouse capture seems
+                //to be released implicitly (even without the left mouse SendMouseClickEvent in leave below)
+                Mouse.Capture(null);
```

Comment:
If we switch to `UIElement.ReleaseMouseCapture();` it has an additional `Mouse.Captured == this` check. 

https://referencesource.microsoft.com/#PresentationCore/Core/CSharp/System/Windows/UIElement.cs,2514

May as well switch to `UIElementCaptureMouse()` as well for consistency.

Reply:
You already made these changes right? It's a little unclear from the commit with the ChromiumWebBrowserWithMouseCapture but it looks like it.

How are you envisioning this drag fix will make it into the next release now that it's experimental? Should I do something? :)

Reply:
> You already made these changes right? It's a little unclear from the commit with the ChromiumWebBrowserWithMouseCapture but it looks like it.

I did. Those changes are only in the `cefsharp/75` branch at the moment. `ChromiumWebBrowserWithMouseCapture` is already marked as `Obsolete`. 

> How are you envisioning this drag fix will make it into the next release now that it's experimental?

I'm reluctant to assign a `milestone` until I get a better understanding of which versions of `Windows 10` the fix will be applied to. Leave this for now, will revisit once there's some clarity. Thanks 👍